csvtotable/convert.py

```diff
@@ -130,9 +130,10 @@ def render_template(table_headers, table_items, **options):
     virtual_scroll = False
     scroll_y = height
 
-    if virtual_scroll_limit != -1 and len(table_items) > virtual_scroll_limit:
-        virtual_scroll = True
-        display_length = -1
+    if virtual_scroll_limit:
```

Comment:
Also variable `virtual-scroll` will defaults to 1000 and you don't need to check if its valid. We just check its its not -1 since -1 disables the virtual scroll.